From 4a689e8eee2451d6a39d851b753d27e7c746d8ba Mon Sep 17 00:00:00 2001
From: KurumiFake <kurumiisshidowife@gmail.com>
Date: Sat, 5 Mar 2022 04:57:55 +0000
Subject: [PATCH] modify default config


diff --git a/flamecord/src/main/java/dev/_2lstudios/flamecord/configuration/FlameCordConfiguration.java b/flamecord/src/main/java/dev/_2lstudios/flamecord/configuration/FlameCordConfiguration.java
index 8c93263b..b0c6c07d 100644
--- a/flamecord/src/main/java/dev/_2lstudios/flamecord/configuration/FlameCordConfiguration.java
+++ b/flamecord/src/main/java/dev/_2lstudios/flamecord/configuration/FlameCordConfiguration.java
@@ -34,7 +34,7 @@ public class FlameCordConfiguration extends FlameConfig {
     @Getter
     private boolean antibotCountryFirewall = true;
     @Getter
-    private Collection<String> antibotCountryBlacklist = Arrays.asList("CN", "HK", "RU", "IN", "TH", "ID", "DZ", "VN", "IR", "PK");
+    private Collection<String> antibotCountryBlacklist = Arrays.asList("CN", "HK", "RU", "IN", "TH", "ID", "DZ", "IR", "PK");
     @Getter
     private boolean antibotCountryLog = true;
     @Getter
diff --git a/proxy/src/main/java/com/mrivanplays/ivancord/conf/IvanCordConfiguration.java b/proxy/src/main/java/com/mrivanplays/ivancord/conf/IvanCordConfiguration.java
index c3934d73..3ef6a379 100644
--- a/proxy/src/main/java/com/mrivanplays/ivancord/conf/IvanCordConfiguration.java
+++ b/proxy/src/main/java/com/mrivanplays/ivancord/conf/IvanCordConfiguration.java
@@ -10,7 +10,7 @@ import net.md_5.bungee.conf.YamlConfig;
 public class IvanCordConfiguration extends WaterfallConfiguration
 {
 
-    private boolean kickToLobby = true;
+    private boolean kickToLobby = false; // AmongCord
     private String brand = "AmongCord"; // AmongCord
 
     @Override
diff --git a/proxy/src/main/java/net/md_5/bungee/conf/YamlConfig.java b/proxy/src/main/java/net/md_5/bungee/conf/YamlConfig.java
index ae4c7ac2..f99e0bc3 100644
--- a/proxy/src/main/java/net/md_5/bungee/conf/YamlConfig.java
+++ b/proxy/src/main/java/net/md_5/bungee/conf/YamlConfig.java
@@ -114,7 +114,7 @@ public class YamlConfig implements ConfigurationAdapter
         Map<String, Object> groups = get( "groups", null );
         if ( groups == null )
         {
-            set( "groups.md_5", Collections.singletonList( "admin" ) );
+            set( "groups.md_5", Collections.singletonList( "default" ) );
         }
     }
 
-- 
2.23.0

