From 37541c4e60164ae66cbf7dfd27a82a49df598e70 Mon Sep 17 00:00:00 2001
From: KurumiFake <kurumiisshidowife@gmail.com>
Date: Sat, 21 May 2022 21:35:40 +0700
Subject: [PATCH] remove kickmessage since its both unused and useless... for
 now


diff --git a/proxy/src/main/java/net/md_5/bungee/connection/DownstreamBridge.java b/proxy/src/main/java/net/md_5/bungee/connection/DownstreamBridge.java
index 08e7c244..8b8eea66 100644
--- a/proxy/src/main/java/net/md_5/bungee/connection/DownstreamBridge.java
+++ b/proxy/src/main/java/net/md_5/bungee/connection/DownstreamBridge.java
@@ -605,9 +605,9 @@ public class DownstreamBridge extends PacketHandler
         ServerInfo defaultServer = bungee.getServerInfo( bungee.getConfig().getListeners().parallelStream().findFirst().get().getDefaultServer() );
         ServerKickEvent event = bungee.getPluginManager().callEvent( new ServerKickEvent( con, server.getInfo(),
                 ComponentSerializer.parse( kick.getMessage() ), def, ServerKickEvent.State.CONNECTED, ServerKickEvent.Cause.SERVER ) );
-        BaseComponent[] kickMessage = new ComponentBuilder( bungee.getTranslation( "server_kick" ) )
+        /*BaseComponent[] kickMessage = new ComponentBuilder( bungee.getTranslation( "server_kick" ) )
                 .append( event.getKickReasonComponent() )
-                .create();
+                .create();*/ // AmongCord
         if ( kickToLobby )
         {
             if ( defaultServer != null )
@@ -615,7 +615,7 @@ public class DownstreamBridge extends PacketHandler
                 con.connectNow( defaultServer, ServerConnectEvent.Reason.KICK_REDIRECT );
                 con.sendMessage( TextComponent.fromLegacyText(
                         bungee.getTranslation( "kick_to_lobby", event.getKickedFrom().getName(),
-                                BaseComponent.toLegacyText( kickMessage ), defaultServer.getName() ) ) );
+                                event.getKickReasonComponent(), defaultServer.getName() ) ) ); // AmongCord
             } else
             {
                 if ( event.isCancelled() && event.getCancelServer() != null )
@@ -623,10 +623,10 @@ public class DownstreamBridge extends PacketHandler
                     con.connectNow( event.getCancelServer(), ServerConnectEvent.Reason.KICK_REDIRECT );
                     con.sendMessage( TextComponent.fromLegacyText(
                             bungee.getTranslation( "kick_to_lobby", event.getKickedFrom().getName(),
-                                    BaseComponent.toLegacyText( kickMessage ), event.getCancelServer().getName() ) ) );
+                                    event.getKickReasonComponent(), event.getCancelServer().getName() ) ) ); // AmongCord
                 } else
                 {
-                    con.disconnect0( kickMessage );
+                    con.disconnect0( event.getKickReasonComponent() ); // AmongCord
                 }
             }
         } else
@@ -636,7 +636,7 @@ public class DownstreamBridge extends PacketHandler
                 con.connectNow( event.getCancelServer(), ServerConnectEvent.Reason.KICK_REDIRECT );
             } else
             {
-                con.disconnect0( kickMessage );
+                con.disconnect0( event.getKickReasonComponent() ); // AmongCord
             }
         }
         // IvanCord end
-- 
2.23.0

