From 1d3ab1b9bf82b2f6cba4133f3523042f0e1b029e Mon Sep 17 00:00:00 2001
From: KurumiFake <kurumiisshidowife@gmail.com>
Date: Fri, 18 Feb 2022 05:49:12 +0000
Subject: [PATCH] rebrand


diff --git a/proxy/src/main/java/net/md_5/bungee/BungeeCord.java b/proxy/src/main/java/net/md_5/bungee/BungeeCord.java
index 75d051b1..82b19985 100644
--- a/proxy/src/main/java/net/md_5/bungee/BungeeCord.java
+++ b/proxy/src/main/java/net/md_5/bungee/BungeeCord.java
@@ -576,7 +576,7 @@ public class BungeeCord extends ProxyServer
     @Override
     public String getVersion()
     {
-        return "0.7.4";
+        return "0.0.2";
     }
 
     public void reloadMessages()
diff --git a/proxy/src/main/java/net/md_5/bungee/command/CommandBungee.java b/proxy/src/main/java/net/md_5/bungee/command/CommandBungee.java
index aeda7963..3d4c6604 100644
--- a/proxy/src/main/java/net/md_5/bungee/command/CommandBungee.java
+++ b/proxy/src/main/java/net/md_5/bungee/command/CommandBungee.java
@@ -17,6 +17,7 @@ public class CommandBungee extends Command
     public void execute(CommandSender sender, String[] args)
     {
         // FlameCord - Use custom version message
-        sender.sendMessage( ChatColor.translateAlternateColorCodes( '&', "&eThis server is running &c" + ProxyServer.getInstance().getName() + "&e version &a" + ProxyServer.getInstance().getVersion() + "&e by &bLinsaFTW & Sammwy&e." ) );
+        // AmongCord - nop no fanci colors xD
+        sender.sendMessage("Running " + ProxyServer.getInstance().getName() + " version " + ProxyServer.getInstance().getVersion() + " by KurumiFake");
     }
 }
diff --git a/query/src/main/java/net/md_5/bungee/query/QueryHandler.java b/query/src/main/java/net/md_5/bungee/query/QueryHandler.java
index 06dcefa2..f64eab15 100644
--- a/query/src/main/java/net/md_5/bungee/query/QueryHandler.java
+++ b/query/src/main/java/net/md_5/bungee/query/QueryHandler.java
@@ -110,8 +110,8 @@ public class QueryHandler extends SimpleChannelInboundHandler<DatagramPacket>
             // Waterfall start
             List<String> players = bungee.getPlayers().stream().map(ProxiedPlayer::getName).collect(Collectors.toList());
 
-            // FlameCord - Rebrand
-            ProxyQueryEvent event = new ProxyQueryEvent(listener, new QueryResult(listener.getMotd(), "SMP", "FlameCord_Proxy",
+            // FlameCord - Rebrand // AmongCord
+            ProxyQueryEvent event = new ProxyQueryEvent(listener, new QueryResult(listener.getMotd(), "SMP", "AmongCord_Proxy",
                     bungee.getOnlineCount(), listener.getMaxPlayers(), listener.getHost().getPort(),
                     listener.getHost().getHostString(), "MINECRAFT",  players, bungee.getGameVersion()));
             QueryResult result = bungee.getPluginManager().callEvent(event).getResult();
-- 
2.20.1

