From d4b5b3817e39a7163883ca8a60906b3dc47918e3 Mon Sep 17 00:00:00 2001
From: KurumiFake <kurumiisshidowife@gmail.com>
Date: Fri, 18 Feb 2022 06:42:28 +0000
Subject: [PATCH] modify def confs

err...

diff --git a/flamecord/src/main/java/dev/_2lstudios/flamecord/configuration/FlameCordConfiguration.java b/flamecord/src/main/java/dev/_2lstudios/flamecord/configuration/FlameCordConfiguration.java
index d0fb0de2..b88250db 100644
--- a/flamecord/src/main/java/dev/_2lstudios/flamecord/configuration/FlameCordConfiguration.java
+++ b/flamecord/src/main/java/dev/_2lstudios/flamecord/configuration/FlameCordConfiguration.java
@@ -54,7 +54,7 @@ public class FlameCordConfiguration {
     @Getter
     private int firewallSeconds = 60;
     @Getter
-    private Collection<String> firewallNames = new HashSet<>(Arrays.asList(new String[] { "mcspam","mcbot","mcstorm","dropbot","theresabot" }));
+    private Collection<String> firewallNames = new HashSet<>(Arrays.asList(new String[] { "mcspam","mcbot","mcstorm","dropbot","theresabot","mcdown" }));
 
     public FlameCordConfiguration(final ConfigurationProvider configurationProvider) {
         try {
diff --git a/proxy/src/main/java/net/md_5/bungee/conf/YamlConfig.java b/proxy/src/main/java/net/md_5/bungee/conf/YamlConfig.java
index ae4c7ac2..adfa874b 100644
--- a/proxy/src/main/java/net/md_5/bungee/conf/YamlConfig.java
+++ b/proxy/src/main/java/net/md_5/bungee/conf/YamlConfig.java
@@ -114,7 +114,7 @@ public class YamlConfig implements ConfigurationAdapter
         Map<String, Object> groups = get( "groups", null );
         if ( groups == null )
         {
-            set( "groups.md_5", Collections.singletonList( "admin" ) );
+            set( "groups.md_5", Collections.singletonList( "default" ) ); // AmongCord
         }
     }
 
-- 
2.35.1

