From 72589ca16eee530eba35339c7a8947aeffd957b5 Mon Sep 17 00:00:00 2001
From: KurumiFake <kurumiisshidowife@gmail.com>
Date: Fri, 20 May 2022 07:18:57 +0700
Subject: [PATCH] log less


diff --git a/proxy/src/main/java/net/md_5/bungee/connection/InitialHandler.java b/proxy/src/main/java/net/md_5/bungee/connection/InitialHandler.java
index f84e6966..87083808 100644
--- a/proxy/src/main/java/net/md_5/bungee/connection/InitialHandler.java
+++ b/proxy/src/main/java/net/md_5/bungee/connection/InitialHandler.java
@@ -455,7 +455,7 @@ public class InitialHandler extends PacketHandler implements PendingConnection
                 {
                     if ( FlameCord.getInstance().getFlameCordConfiguration().isAntibotRatelimitLog() )
                     {
-                        bungee.getLogger().log( Level.INFO, "[FlameCord] [{0}] is pinging too fast", ch.getRemoteAddress() );
+                        bungee.getLogger().log( Level.INFO, "{0} is pinging too fast", ch.getRemoteAddress() );
                     }
 
                     disconnect( bungee.getTranslation( "antibot_ratelimit", addressData.getPingsSecond() ) );
@@ -481,7 +481,7 @@ public class InitialHandler extends PacketHandler implements PendingConnection
                 {
                     if ( FlameCord.getInstance().getFlameCordConfiguration().isAntibotRatelimitLog() )
                     {
-                        bungee.getLogger().log( Level.INFO, "[FlameCord] [{0}] is connecting too fast", ch.getRemoteAddress() );
+                        bungee.getLogger().log( Level.INFO, "{0} is connecting too fast", ch.getRemoteAddress() );
                     }
 
                     disconnect( bungee.getTranslation( "antibot_ratelimit", addressData.getConnectionsSecond() ) );
@@ -537,7 +537,7 @@ public class InitialHandler extends PacketHandler implements PendingConnection
         {
             if ( FlameCord.getInstance().getFlameCordConfiguration().isAntibotAccountsLog() )
             {
-                bungee.getLogger().log( Level.INFO, "[FlameCord] [{0}] has too many accounts", ch.getRemoteAddress() );
+                bungee.getLogger().log( Level.INFO, "{0} has too many accounts", ch.getRemoteAddress() );
             }
 
             disconnect( bungee.getTranslation( "antibot_accounts", addressData.getNicknames().size() ) );
@@ -548,7 +548,7 @@ public class InitialHandler extends PacketHandler implements PendingConnection
         {
             if ( FlameCord.getInstance().getFlameCordConfiguration().isAntibotNicknameLog() )
             {
-                bungee.getLogger().log( Level.INFO, "[FlameCord] [{0}] has a blacklisted nickname", ch.getRemoteAddress() );
+                bungee.getLogger().log( Level.INFO, "{0} has blacklisted nickname", ch.getRemoteAddress() );
             }
 
             disconnect( bungee.getTranslation( "antibot_nickname", loginRequest.getData() ) );
@@ -559,7 +559,7 @@ public class InitialHandler extends PacketHandler implements PendingConnection
         {
             if ( FlameCord.getInstance().getFlameCordConfiguration().isAntibotReconnectLog() )
             {
-                bungee.getLogger().log( Level.INFO, "[FlameCord] [{0}] has to reconnect to join", ch.getRemoteAddress() );
+                bungee.getLogger().log( Level.INFO, "{0} has to reconnect to join", ch.getRemoteAddress() );
             }
 
             disconnect( bungee.getTranslation( "antibot_reconnect", FlameCord.getInstance().getFlameCordConfiguration().getAntibotReconnectAttempts() - addressData.getTotalConnections() ) );
@@ -570,7 +570,7 @@ public class InitialHandler extends PacketHandler implements PendingConnection
         {
             if ( FlameCord.getInstance().getFlameCordConfiguration().isAntibotCountryLog() )
             {
-                bungee.getLogger().log( Level.INFO, "[FlameCord] [{0}] has his country blocked from the server", ch.getRemoteAddress() );
+                bungee.getLogger().log( Level.INFO, "{0} is country blocked", ch.getRemoteAddress() );
             }
 
             disconnect( bungee.getTranslation( "antibot_country", addressData.getCountry() ) );
-- 
2.23.0

